libs/utils/utils.js

```diff
@@ -592,14 +592,19 @@ export async function loadDeferred(area, blocks, config) {
   });
 }
 
-function loadPrivacy() {
+export function loadPrivacy() {
   window.fedsConfig = {
     privacy: {
       otDomainId: '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
-      footerLinkSelector: '[href="https://www.adobe.com/#openPrivacy"]',
```

Comment:
I think instead of adding our own event listener, we just need to fix the footerLinkSelector to something like `footerLinkSelector: '[href*="#openPrivacy"]',`
This fixes the issue with the modal not opening - 
https://main--bacom--adobecom.hlx.page/kr/customer-success-stories?milolibs=vhargrave-privacy-test&martech=off

The link text is then being translated by the library, but maybe there's a config option we can pass so this doesn't happen. @narcis-radu & @overmyheadandbody - is something like that possible?

Reply:
@vhargrave - there's no option to skip translation, the only feature we have is to use page language (instead of browser language). If no `footerLinkSelector` is provided (or the associated DOM element can't be found), then the event listener has to be set manually, just like @meganthecoder did.
Since this is implemented in Milo, I would like to go one step further: instead of adding an event listener to a DOM element, let's listen to a hash change and allow the Preference Center modal to be opened from the page content as well. This will get used on pages like https://www.adobe.com/privacy/opt-out.html.

Reply:
@narcis-radu I prefer attaching the listener to the DOM element for performance reasons. Milo uses hashes for modals which are fairly popular content-wise. I'd rather not do a check for openPrivacy every time, especially  because we don't have a specific need to open the modal from anywhere other than the footer.

Reply:
makes sense, go for it!